src/utils/removeObjectUndefinedValue.js

```diff
@@ -0,0 +1,7 @@
+function removeObjectUndefinedValue(object) {
+  Object.keys(object).forEach(key => !object[key] && delete object[key]);
```

Comment:
check explicitly for undefined, to not remove null-s for example

if (object[key] === undefined) {
    delete object[key];
}